study.md

```diff
@@ -40,14 +41,19 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It says that I want to use again the array that I have created before
```

Comment:
The `!` at the end of a method name signifies that it modifies the object that it was invoked upon rather than returning a modified copy of it.